code/modules/mining/abandonedcrates.dm

```diff
@@ -23,16 +23,10 @@
 /obj/structure/closet/crate/secure/loot/proc/generate_loot()
 	var/loot = rand(1, 100)
 	switch(loot)
-		if(1 to 5) // Common things go, 5%
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/rum(src)
-			new/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)
-			new/obj/item/weapon/reagent_containers/food/snacks/grown/ambrosiadeus(src)
-			new/obj/item/weapon/flame/lighter/zippo(src)
-		if(6 to 10)
-			new/obj/item/weapon/pickaxe/drill(src)
-			new/obj/item/device/taperecorder(src)
-			new/obj/item/clothing/suit/space(src)
-			new/obj/item/clothing/head/helmet/space(src)
+		if(1 to 10)
```

Comment:
I think you are overdoing it a bit, you increase the dropchance tenfold.
I believe a 5% drop chance would be fine (6 to 10)
Might also be a good idea to just spawn one KA inside of the crate, instead of 3
If science is indeed unreliable, then we can explore adding medium tier KAs / KA Parts to cargo

Reply:
I keep forgetting to message you about this.

Yeah, I'll drop the rate down to 5%, and make some of them orderable.